pkg/pillar/cmd/nim/nim.go

```diff
@@ -1188,3 +1199,61 @@ func isAssigned(ctx *nimContext, ifname string) bool {
 	}
 	return false
 }
+
+const (
+	configDPCDir = types.IdentityDirname + "/DevicePortConfig"
```

Comment:
NIT: This `DPC` abreviation is a bit hard to read, I would rather go with `DevicePortConfigDir`, which is not a lot longer, but a lot more understandable.
Same applies for `dpcl`, `runDPCDir`, etc

Reply:
Elsewhere in EVE there is `DeviceNetworkStatus` abbreviated to DNS and in the same package there are also references to DNS as the protocol, so it gets even more confusing. I would also prefer longer names like `DevicePortConfigDir` or alternatively a bit shorter one `DevPortConfDir`, but not `DPC`.

Reply:
I know the DNS acronym is confusing; our only defense is that the DomainNameSystem code and DeviceNetworkStatus code was earlier quite separate.
Will spell out this DPC stuff.